tools/terser.d.ts

```diff
@@ -96,7 +96,6 @@ export interface ManglePropertiesOptions {
 
 export interface FormatOptions {
     ascii_only?: boolean;
-    beautify?: boolean;
```

Comment:
This will break TS compilation if people want to use the old-style `beautify` option as anything other than an alias.

Reply:
We can add `/** @deprecation Not implemented anymore. */` which will be highlighted (striked through) in some editors and can be warned with `eslint-typescript`.

Reply:
Nice suggestion! I didn't know about that

Reply:
Added a deprecation annotation as suggested by @HolgerJeromin